lib/actions/jira.ts

```
@@ -0,0 +1,565 @@
+ */
+export async function getJiraTicketAction(
+  identifier: string,
+  isIssueKey = false
+): Promise<ActionResponse<any>> {
+  try {
+    const session = await auth();
+    if (!session?.user) {
+      return { success: false, error: 'Unauthorized' };
+    }
+
+    let ticket;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)